packages/app-provider/src/ApolloContainer.tsx

```diff
@@ -0,0 +1,127 @@
+import * as React from 'react';
+import * as R from 'ramda';
+import { ApolloProvider } from 'react-apollo';
+import { InMemoryCache, IntrospectionFragmentMatcher } from 'apollo-cache-inmemory';
+import { AuthClient, Authorizable, AuthState } from '@8base/utils';
+import { ApolloContainerPassedProps } from './types';
+import { FragmentsSchemaContainer } from './FragmentsSchemaContainer';
+
+const { EightBaseApolloClient } = require('@8base/apollo-client');
+const { withAuth } = require('@8base/auth');
```

Comment:
@ej9x Why do you use `require` here?

Reply:
To avoid type checking. because still has no typings for these packages